products/kotlin.md

```diff
@@ -58,4 +53,9 @@ releases:
 > Kotlin is designed to interoperate fully with Java, and the JVM version of Kotlin's standard library depends on the Java Class Library,
 > but type inference allows its syntax to be more concise. Kotlin mainly targets the JVM, but also compiles to JavaScript
 > (e.g., for frontend web applications using React) or native code (via LLVM); e.g., for native iOS apps sharing business logic with Android apps.
-> Language development costs are borne by JetBrains, while the Kotlin Foundation protects the Kotlin trademark.
+  
+Only the latest version sees active development and gets security fixes.
```

Comment:
Maybe we could be a bit more explicit about the lack of information?

I mean something like :

> Kotlin support policy [is not clearly defined](https://discuss.kotlinlang.org/t/kotlin-support-roadmap/11454). But usually only the latest version sees active development and [gets bug and security fixes](https://kotlinlang.org/docs/kotlin-evolution.html#dealing-with-compiler-bugs).